pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1775,6 +1776,18 @@ func buildHTTPConnectionManager(pluginParams *plugin.InputParams, env *model.Env
 		filters = append(filters, &http_conn.HttpFilter{Name: wellknown.GRPCWeb})
 	}
 
+	if IsIstioVersionGE14(pluginParams.Node) &&
+		pluginParams.ServiceInstance.Endpoint.ServicePort.Protocol == protocol.GRPC {
```

Comment:
do we want this always enabled? My understanding/guess is this has no impact for telemetry v1 and makes grpc work for telemetry v2?

Reply:
this will make envoy emit native stats for grpc. we can then leverage it for telemetry v2 (e.g. grpc message counts).

Reply:
Ok so since we exclude all the stats without v2, this will have no negative impact for users on v1?

Reply:
There is an impact on CPU but I don't have a way to measure it easily.

Reply:
How big will the impact be? is it just the envoy level additional filter for recording stats per grpc connection or is it far more than that? Also, what is the memory hit?

Reply:
CPU impact is hard to tell. On synthetic stress test up to 5% which is similar to what most extensions cost. Memory overhead is tiny.